src/features/Navigation/InfoDropdown/DropdownButton.tsx

```diff
@@ -20,17 +20,17 @@ export const DropdownButton: React.FC<Props> = ({
   text,
 }) => {
   const { ref, isHovering } = useHover();
-  const color = isHovering || isComponentVisible ? 'darkblue' : 'white';
+  const color = isHovering || isComponentVisible ? 'blueDark' : 'white';
 
   return (
     <Button
       isExpanded={isComponentVisible}
       onClick={() => setIsComponentVisible(!isComponentVisible)}
       ref={ref}
     >
-      <Text variant={isHovering ? 'linkDisabled' : 'link'} color={color}>
+      <ButtonText variant="link" color={color}>
         {text}
-      </Text>
+      </ButtonText>
       {isComponentVisible ? (
         <ChevronUp size={8} color={color} />
       ) : (
```

Comment:
For some reason now the when hovering the NavigationBars Dropdown-buttons, (for info-dropdown and navigation-items), the icon becomes invisible.
It works in current master. Dont know really what causes it, because I tried to debug it a bit.

Reply:
Additionally when in mobile-mode, the Icon is now not centered, but in a weird position in the top

Reply:
Now I got why the color is not working on the icon anymore. See the svg-color-property when I am hovering =>
<img width="635" alt="image" src="https://user-images.githubusercontent.com/34128180/214937998-0d1da0d4-118f-40b8-a339-68378e7172ae.png">

Reply:
So the old color, darkBlue worked. Because its an actual CSS color, lol

Reply:
Ah yeah we try to fill it with the string 'blueDark' and not the acutal color code 😄 So this was actually an old bug that you found! Nice find! And the 'white' of course also works because it's a CSS color too! 😆

Reply:
Yeah, so the ChevronUp - and ChevronDown -compoennts need to map the palette with the key, easy fix

But the other bug, is new. => "in mobile-mode, the Icon is now not centered, but in a weird position in the top"